src/app/testing/jest/frontend/components/cards/CollegeCard.test.tsx

```diff
@@ -44,8 +44,8 @@ describe("Test CollegeCard", () => {
     );
   });
 
-  test("CollegeCard button group should have one button", () => {
-    expect(screen.getAllByRole("button")).toHaveLength(1);
+  test("CollegeCard button group should have two buttons", () => {
```

Comment:
Need to revert this test as well to one button for the time being

Reply:
Reverted